src/pyspiffe/__init__.py

```diff
@@ -0,0 +1 @@
+SPIFFE_SCHEME = 'spiffe'
```

Comment:
s/SPIFFE_SCHEME/SPIFFE_SCHEMA

Reply:
why `SCHEMA` ?  The URI spec uses `SCHEME`: https://tools.ietf.org/html/rfc3986#page-17

Reply:
if SCHEME is the right word we should then standarize on it, since we also have [`SCHEMA_SUFFIX`](https://github.com/HewlettPackard/py-spiffe/pull/3/files#diff-b3c9101ea47eaa2dcd58b5603cd29f76d090b1f4c5413272a97bffad4ffcdc07R6)

Reply:
yes, you are right, will make it consisteng